ddp-workspace/projects/ddp-circadia/app.yaml

```diff
@@ -1,5 +1,5 @@
 service: circadia
```

Comment:
not active

Reply:
All inactive studies do not deploy except `circadia`. Are you okay with combine unrelated change in PR?

Reply:
yes alex. circadia is inactive too